CHANGELOG.md

```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Handle `<base href="...">` values correctly when inlining imports.
 - Apply `<base target="...">` to links and forms in the same document as
   appropriate.
+- Fixed a bug where element attributes inside `<template>` tags were not
```

Comment:
Prefer either doing parallel PRs or using github's base branch feature to chain PRs off of one another. Otherwise it's tricky to tell in which PR I should leave a particular comment.